lib/lita/handlers/pagerduty_utility.rb

```diff
@@ -96,7 +96,7 @@ def forget(response)
       private
 
       def lookup_on_call_user(schedule_id)
-        now = Time.now.utc
+        now = Time::Time.now.utc
```

Comment:
It appears that you also have the lita-time handler installed, which is causing a namespace conflict. In this case, I believe the canonical way to reference a top level constant is to prefix it with `::`. Could you change this line to `now = ::Time.now.utc` and see whether it works for you?